### DISCUSSION TITLE
Python (Flask) + Next example project fails to build on Vercel

### DISCUSSION THREAD

**Discussion by shershen08** [vercel/community#7458] [Help]
### Summary

example project - https://vercel.com/templates/next.js/nextjs-flask-starter
when creating a copy and running a build fails with 
```
Error: Command failed: pip3.12 install --disable-pip-version-check --target . --upgrade -r /vercel/path0/requirements.txt
ERROR: Exception:
```

<img width="1194" alt="Screenshot 2024-07-13 at 12 47 45" src="https://github.com/user-attachments/assets/d7ede79c-6f41-4ac9-be97-16fad6c59962">

### Steps to Reproduce

clone copy of https://vercel.com/templates/next.js/nextjs-flask-starter

**Comment by pawlean** [comment]
Thanks for reporting, @shershen08!

You're right, I get the same issue. It looks like there's an issue with `requirements.txt` - perhaps an outdated dependency. I'll see if I can raise a fix later. 😄 

Feel free to clone the repo and make the changes yourself if you'd like!

[a comment by an automated account is not shown]
